src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/CqlProxyRetryPolicy.java

```diff
@@ -64,16 +62,19 @@ public RetryDecision onWriteTimeout(
       int blockFor,
       int received,
       int retryCount) {
+    var retryDecision = RetryDecision.RETHROW;
     if (retryCount < MAX_RETRIES && writeType == WriteType.CAS) {
-      if (LOG.isInfoEnabled()) {
-        LOG.info(
-            "Retrying write timeout for request type : {}  with retryCount: {}",
-            writeType,
-            retryCount + 1);
-      }
-      return RetryDecision.RETRY_SAME;
+      retryDecision = RetryDecision.RETRY_SAME;
     }
-    return RetryDecision.RETHROW;
+    if (LOG.isInfoEnabled()) {
+      LOG.info(
+          "Write timeout for request write type : {}, retryCount: {}, consistency level: {} -> retry decision is: {}",
+          writeType,
+          retryCount + 1,
```

Comment:
I know we used to do "+1" but I don't think that should be done here -- it used to indicate that the retry to be done has that count, but here we are indicating the current count. So let's just print `retryCount` as given.

Reply:
nit: "write type" -> "writeType" (name of argument given)

Reply:
thanks!